src/components/calculator.js

```diff
@@ -0,0 +1,67 @@
+import TheScreen from './screen';
```

Comment:
![image](https://github.com/ometman/math-magicians-app/assets/93039359/58e08325-a69a-4f84-ba44-2902515657f0)

- [ ] It would be nice if you start the names of your components with a Capital letter means calculator.js should be Calculator.js and screen.js should be Screen.js because of two reasons ⬇️ 👀

1. First, React has a naming convention in which you always name a component with a capital letter and whenever you create a logic file or some code which is directly not meant for the frontend then that file should have a name with small letters 🤔
2. Second, it would be nice if you renamed them because in your project requirements your Calculator.js component starts with a capital letter 👍

Reply:
Good reminder. 👍 
Fixed.